### PR TITLE
[doc] add glossary anchor

### PR DESCRIPTION
**Pull request by Asu4ni**
related #340 
add glossary anchor
little fix: move images into folder and change display size

### CODE REVIEW

**Comment by Asu4ni** [comment]
not finish yet

**Comment by Asu4ni** [comment]
done

**Comment by Asu4ni** [comment]
I'll squash all commits to one after others have examined them.

File: doc/source/glossary.rst [hunk 1]
```diff
@@ -7,38 +7,42 @@ Glossary
 
    Resource
      An entity representing anything in your infrastructure that you will
-     associate metric(s) with. It is identified by a unique ID and can
+     associate |metric|\ (s) with. It is identified by a unique ID and can
```
Comment (chungg) [inline doc/source/glossary.rst]: should there be a random space between \ and (s)

Comment (Asu4ni) [reply doc/source/glossary.rst]: that is to make | separated with (, but there should not be space there. Putting a \ before it makes it a functional space.

**Pull request merged**